src/components/auth/Input/index.tsx

```diff
@@ -1,27 +1,73 @@
-import { FieldError, RegisterOptions, useFormContext } from "react-hook-form";
+import { FieldError, RegisterOptions, useFormContext } from 'react-hook-form';
+import * as Styled from './Input.styled';
+import Cancel from '@/components/@common/SVG/Icon/Cancel';
 
 export interface IInputFieldProps
   extends React.InputHTMLAttributes<HTMLInputElement> {
   label?: string;
+  /** label의 typo 타입을 지정하기 위함. (true: T2_B, false: B1_B) */
+  boldLabel?: boolean;
+  /** input 요소가 필수값인지 확인하기 위함 */
+  isMandatory?: boolean;
```

Comment:
```suggestion
  isRequired?: boolean;
```
필수적인 값을 사용할 때, `isRequired`를 많이 사용한 기억이 있어 남겨봅니다☺️

Reply:
에러 메세지 처리할 때도 필수항목에 `required`를 사용했으니 `required`가 더 좋을 것 같습니다! 감사합니다 👍